materialize-redshift/driver.go

```diff
@@ -935,6 +943,8 @@ func (d *transactor) commit(ctx context.Context, fenceUpdate string, hasUpdates
 		}
 	}
 
+	log.Info("store: finished encoding and uploading of files")
```

Comment:
Above here, where we run individual copy & merge queries, I think we should be logging around each one along the lines of "starting merging into table <TABLE>" & "finished merged into table <TABLE>", replacing "merging" with "copying" as applicable. It would be nice to know which table is taking the longest to load data into.